webpack.dev.js

```diff
@@ -27,7 +27,10 @@ module.exports = merge(common, {
     hotOnly: true,
     port: 7000,
     host: 'localhost',
-    historyApiFallback: true,
+    historyApiFallback: {
+      index: '/',
+      disableDotRule: true,
+    },
     // used for app catalogs proxy /catalogs?url=
```

Comment:
This is so paths with dots in them work in development (e.g. `/apps/control-plane-catalog/api/1.29.0`)